openml/_api_calls.py

```diff
@@ -102,12 +103,39 @@ def _read_url(url, data=None):
     return response.text
 
 
+def send_request(request_method, url, kwargs):
```

Comment:
i hate kwargs, it's so vague. Do we need this level of flexibility or can we just add optional arguments?

Reply:
Changed.